src/v2/Components/StickyContainer.tsx

```diff
@@ -36,7 +44,14 @@ const Sentinel = styled(Box).attrs({
   height: 0;
 `
 
-export const StickyContainer: React.FC = ({ children }) => {
+export interface StickyContainerProps {
+  ContainerComponent?: React.ComponentType<BaseContainerProps>
+}
+
+export const StickyContainer: React.FC<StickyContainerProps> = ({
+  children,
+  ContainerComponent = DefaultContainer,
+}) => {
```

Comment:
`non-blocking`: As an alternative I could see this container having no opinions about styling outside of what's needed to stick at the correct position, since the border variant here is pretty specific to filters rather than being a 'default'. Then use a context provider + hook to pull out `stuck` status in the child.

Reply:
Agree. Is it okay if I add new PR for these changes?

Reply:
👍 👍